src/ServiceControl/Operations/ImportFailedErrors.cs

```diff
@@ -5,69 +5,84 @@
     using System.Threading.Tasks;
     using NServiceBus.Extensibility;
     using NServiceBus.Logging;
+    using NServiceBus.Raw;
     using NServiceBus.Transport;
     using Raven.Client;
+    using Recoverability;
 
     class ImportFailedErrors
     {
-        public ImportFailedErrors(IDocumentStore store, ErrorIngestor errorIngestor)
+        public ImportFailedErrors(IDocumentStore store, ErrorIngestor errorIngestor, RawEndpointFactory rawEndpointFactory)
         {
             this.store = store;
             this.errorIngestor = errorIngestor;
+            this.rawEndpointFactory = rawEndpointFactory;
         }
 
-        public async Task Run(CancellationTokenSource tokenSource)
+        public async Task Run(CancellationToken token)
         {
-            var token = tokenSource.Token;
-            var succeeded = 0;
-            var failed = 0;
-            using (var session = store.OpenAsyncSession())
+            var config = rawEndpointFactory.CreateSendOnlyRawEndpointConfiguration("ImportFailedErrors");
```

Comment:
I'm assuming this name is arbitrary as we shouldn't be changing any headers or anything.

Reply:
Yes. For a send-only endpoint this name is arbitrary.